uploader.json

```diff
@@ -1826,6 +1826,16 @@
             "file": "/var/log/redhat_access_proactive/redhat_access_proactive.log",
             "pattern": [],
             "symbolic_name": "redhat_access_proactive_log"
+        },
+        {
+            "file": "/etc/logrotate.conf",
+            "pattern": [],
+            "symbolic_name": "logrotate_conf"
+        },
+        {
+            "file": "/etc/logrotate.d/()*",
+            "pattern": [],
+            "symbolic_name": "logrotate_d"
```

Comment:
@xiangce your regex needs to be changed in order to collect anything, like this: `"/etc/logrotate.d/().*"`.  Note the added `.`. Also since there is only one spec being evaluated by the parser, you should use the same symbolic name, `logrotate_conf` for both entries here.  All of the entries collected will be merged into the one `Spec.logrotate_conf` spec.

Reply:
Thank you @bfahr, I updated as per your suggestion.